### PR TITLE
Add ACPR tuto slides and news

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

File: _posts/2019-11-26-ACPR-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  ACPR Tutorial on Digital Geometry in Pattern Recognition
+---
+
+The presentations of the two tutorial on Digital Geometry in Pattern Recognition are available on the [Ressource section](https://dgtal.org/documentation/):
```
Comment (dcoeurjo) [inline _posts/2019-11-26-ACPR-tutorial.md]: Tutorials

**Review by dcoeurjo — COMMENTED** [review]
Just a typo...

File: _posts/2019-11-26-ACPR-tutorial.md (hunk 1 above)
Comment (kerautret) [reply _posts/2019-11-26-ACPR-tutorial.md]: thanks, done ;)

**Comment by dcoeurjo** [comment]
Thanks

**Comment by dcoeurjo** [comment]
Ok it seems that there are issues with the post: images are not ok and it does not appear as the latest post (date issue ?).

**Comment by dcoeurjo** [comment]
I have fixed the date (and remaining typo). Can you please fix the images ?

**Comment by kerautret** [comment]
I look it sorry ;(

**Comment by kerautret** [comment]
> I have fixed the date (and remaining typo). Can you please fix the images ?

I push directly on the master

**Comment by kerautret** [comment]
@dcoeurjo done ;)

**Comment by dcoeurjo** [comment]
ok thx I've slightly update the text

**Comment by kerautret** [comment]
thanks 👍🏻

**Pull request merged**